presubmit-service/check_if_pr_can_cherry_pick.sh

```diff
@@ -0,0 +1,67 @@
+#! /bin/bash
+
+# The organization containing the repo to be checked against, e.g. angular
+org=$1
+# The repo to be checked against, e.g. material2
+repo=$2
+# The PR number to retrieve the patch files for, e.g. 1234
+pr=$3;
+# The branch in the repo to check for cherry-picking onto, e.g. 6.1.x
+branch=$4;
+# The fullname of the repository to be checked against, e.g. angular/material2
+full_repo="${org}/${repo}"
+# Whether the PR can be cherry-picked on to the targetted branch.
+result=false;
+
+# If the needed tmp directories don't exist, create them.
+[ ! -d "/tmp/presubmit_service/$full_repo/" ] && mkdir -p /tmp/presubmit_service/$full_repo;
+
+# If the local_repo directory does not exist, create it.
+[ ! -d ".presubmit_service/local_repo" ] && mkdir -p .presubmit_service/local_repo;
+cd .presubmit_service/local_repo;
```

Comment:
Did we come to a conclusion with that? Right now, this is always created relatively to the working directory, but I think this should be stored in the user-profile under `$HOME`. Thoughts?

Reply:
Probably something to handle in the future though. Just feels a bit odd to store it relatively to the working directory (meaning that there can be a `local_repo` multiple times; depending on where the script has been launched)

Reply:
The application is always run from `/app`.  So since it always run from the same location I think its fine.